coreos-devel/sdk-depends/sdk-depends-0.0.1.ebuild

```diff
@@ -47,6 +47,7 @@ DEPEND="
 	sys-firmware/edk2-ovmf
 	sys-fs/btrfs-progs
 	sys-fs/cryptsetup
+	dev-perl/Parse-Yapp
```

Comment:
I still don't get it why we need to add some deps here and not let the ebuild pull those deps.

Reply:
Same as above, we need that perl lib in the SDK to build the Samba package. "SDK" is a different target than "base-image" and Gentoo has no concept we could use to map this.